starfish.js

```diff
@@ -123,14 +125,28 @@ function isContributionInTimeRange(createdAt, startMoment, endMoment) {
     );
 }
 
+function filterResponseFor(organization) {
+    if (githubOrganizations.length && githubOrganizations[0] !== '') {
+        return githubOrganizations.indexOf(organization) >= 0;
```

Comment:
I tested this PR out locally. Ignoring a repository works, but I couldn't get ignoring an organization to work. I think maybe the code here isn't working correctly? 

If you're up for it, you might want to write some tests. If not, you can just test your code locally using a sample CSV that includes
- someone who has a PR in the org and repo you're testing, and at least one PR somewhere else
- someone with PRs that are only in the org and repo you're testing (you can use me for `indeedeng/starfish`)
- someone who has a PR in the timeframe, but not in the org and repo you're testing (I think this one might be broken right now, but I'm not sure)
Any other cases I haven't thought of that should be in there?

Try running the unchanged code and your code changes with your CSV and make sure that they match.

Reply:
You're right, I created the env as ORGANIZATION_GITHUB and was considering ONLY the ones that were in the env. However, if the behavior is to ignore the env organizations, the logic was actually reversed :face_with_head_bandage: 

But now I fixed this :laughing:

Reply:
I don't know if I'm prepared to create tests specifically for these scenarios, but I manually tested your examples